pkg/web_app/lib/src/account.dart

```diff
@@ -280,9 +308,43 @@ class _CreatePublisherWidget {
     }
   }
 
-  bool get isActive =>
-      _unauthenticatedRoot != null &&
-      _authenticatedRoot != null &&
-      _publisherIdInput != null &&
-      _createButton != null;
+  bool get isActive => _publisherIdInput != null && _createButton != null;
+}
+
+class _PublisherAdminWidget {
```

Comment:
Could we write a doc-comment saying where this is active, this would make it slightly easier to track..